sim/target-agnostic.mk

```diff
@@ -71,16 +73,16 @@ $(VERILOG) $(HEADER) $(fame_annos): $(FIRRTL_FILE) $(ANNO_FILE) $(SCALA_BUILDTOO
 # to generate a runtime configuration that is compatible with the generated
 # hardware (BridgeModule). Useful for modelling a memory system that differs from the default.
 .PHONY: conf
-conf: $(fame_annos)
+conf: $(fame_annos) $(FAT_JAR)
 	mkdir -p $(GENERATED_DIR)
 	# Runtime configuration generator must run under SBT currently; When
 	# launched via bloop some Console input and output is lost.
-	cd $(base_dir) && $(SBT) "project $(firesim_sbt_project)" "runMain midas.stage.RuntimeConfigGeneratorMain \
+	cd $(base_dir) && java $(JAVA_ARGS) -cp $(FAT_JAR) midas.stage.RuntimeConfigGeneratorMain \
```

Comment:
Looking at this again, if someone expects `make ENABLE_BLOOP=1 conf` to work, my change here will break that.  We could just roll this target back to be hardcoded for SBT.

Reply:
Defaulting to SBT seems reasonable.

Reply:
Rolled this back in 9f611c63d